lib/otel/span-processor.js

```diff
@@ -254,4 +272,26 @@ module.exports = class NrSpanProcessor {
     }
     this.#reconciler.reconcile({ segment, otelSpan: span, mapper })
   }
+
+  reconcileLambdaAttributes({ segment, span }) {
+    // see https://github.com/open-telemetry/opentelemetry-specification/blob/v1.7.0/specification/trace/semantic_conventions/faas.md#example
+    // Add entity linking attrs: cloud.resource_id and cloud.platform
+    segment.addAttribute('cloud.platform', 'aws_lambda')
+    const region = span.attributes['cloud.region'] ?? span.attributes['faas.invoked_region'] ?? 'unknown'
```

Comment:
i don't see region on these spans. you can extract region and account id from `faas.id`